File: APP_ENGINE/BlueCompass/src/main/java/com/singhtwenty2/OceanVista/data/model/dto/response/BeachResponseDTO.java
package com.singhtwenty2.OceanVista.data.model.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalTime;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class BeachResponseDTO {
    private Long id;
    private String name;
    private double latitude;
    private double longitude;
    private String region;
    private LocalTime openingTime;
    private LocalTime closingTime;
    private List<String> activities;
    private List<String> photos;
}

File: NATIVE_ANDROID/app/src/main/java/com/singhtwenty2/oceanvista/feature_home/data/remote/model/dto/response/BeachResponseDTO.kt
package com.singhtwenty2.oceanvista.feature_home.data.remote.model.dto.response

data class BeachResponseDTO(
    val id: Int,
    val name: String,
    val latitude: Double,
    val longitude: Double,
    val region: String,
    val openingTime: String,
    val closingTime: String,
    val activities: List<String>,
    val photos: List<String>
)
